.circleci/config.yml

```diff
@@ -0,0 +1,31 @@
+version: 2.1
```

Comment:
So this wont pass until you have a membership. If another option is cheaper, lets use that.

Reply:
Let's just use Travis since it's free for open-source, the travis.yml file on the develop branch will build against macOS.  I had to remove the WebSocket test because their macOS 10.15 environment isn't supported yet.